experimental/Permutations/Permutations.jl

```diff
@@ -0,0 +1,136 @@
+# This file implements a new way to input permutations in Julia. For example
+# it is possible to create a permutation as follow
+# pi = Oscar.Permutations.@perm (1,2,3)(4,5)(6,7,8)
+# > (1,2,3)(4,5)(6,7,8)
+# For this we use macros to modify the syntax tree of (1,2,3)(4,5)(6,7,8) such that
+# Julia can deal with the expression.
+
+module Permutations
+
+ using GAP
+ using Oscar
```

Comment:
Scratch that and put it into `src/Groups/perm.jl` directly into the top module :-)